.clang-format

```diff
@@ -133,6 +133,9 @@ MaxEmptyLinesToKeep: 1
 Macros:
 - _CCCL_TEMPLATE(...)=template<...>
 - _CCCL_REQUIRES(...)=requires (...)
+- _CCCL_TRY(...)=try { __VA_ARGS__ }
+- _CCCL_CATCH_OR(...)=catch __VA_ARGS__
+- _CCCL_CATCH(...)=catch __VA_ARGS__
```

Comment:
I believe it would be simpler if we would add them to either `StatementMacros` or even better to [IfMacros](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#ifmacros)

Reply:
i don't think `IfMacros` is appropriate because the assumption with `IfMacros` is that the body is _outside_ of the macro. for these macros, the body is inside the macro.

i'll give `StatementMacros` a shot.